brandmateriel/src/game.py

```diff
@@ -636,7 +636,7 @@ def do_step(self, surface):
             try:
                 self._gameover = (
                     self._gameover_timer
-                    < dt.datetime.now() - dt.timedelta(seconds=1)
+                    < dt.datetime.now() - dt.timedelta(seconds=10)
```

Comment:
Why this change? (I honestly can't remember what the `_game_over_timer` does...)

Reply:
I don't remember either. My guess that's the play time after player death.

Reply:
Interesting. I wonder why it does that. I probably put that timer in place because I couldn't figure it out...

Reply:
I'll revert this.

Reply:
Ah, you've already done it. Thanks!